db/migrate/20220525122759_init_schema.rb

```diff
@@ -0,0 +1,237 @@
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.index %w[organization_id code], name: 'index_add_ons_on_organization_id_and_code', unique: true
+      t.index ['organization_id'], name: 'index_add_ons_on_organization_id'
+    end
+
+    create_table 'applied_coupons', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
+      t.uuid 'coupon_id', null: false
+      t.uuid 'customer_id', null: false
+      t.integer 'status', default: 0, null: false
+      t.integer 'amount_cents', null: false
+      t.string 'amount_currency', null: false
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.datetime 'terminated_at', precision: nil
+      t.index %w[coupon_id customer_id], name: 'index_applied_coupons_on_coupon_id_and_customer_id', unique: true, where: '(status = 0)'
+      t.index ['coupon_id'], name: 'index_applied_coupons_on_coupon_id'
+      t.index ['customer_id'], name: 'index_applied_coupons_on_customer_id'
+    end
+
+    create_table 'billable_metrics', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
+      t.uuid 'organization_id', null: false
+      t.string 'name', null: false
+      t.string 'code', null: false
+      t.string 'description'
+      t.jsonb 'properties', default: {}
+      t.integer 'aggregation_type', null: false
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.string 'field_name'
+      t.index %w[organization_id code], name: 'index_billable_metrics_on_organization_id_and_code', unique: true
+      t.index ['organization_id'], name: 'index_billable_metrics_on_organization_id'
+    end
+
+    create_table 'charges', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
+      t.uuid 'billable_metric_id'
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.uuid 'plan_id'
+      t.string 'amount_currency', null: false
+      t.integer 'charge_model', default: 0, null: false
+      t.jsonb 'properties', default: '{}', null: false
+      t.index ['billable_metric_id'], name: 'index_charges_on_billable_metric_id'
+      t.index ['plan_id'], name: 'index_charges_on_plan_id'
+    end
+
+    create_table 'coupons', id: :uuid, default: -> { 'gen_random_uuid()' }, force: :cascade do |t|
+      t.uuid 'organization_id', null: false
+      t.string 'name', null: false
+      t.string 'code'
+      t.integer 'status', default: 0, null: false
+      t.datetime 'terminated_at'
+      t.bigint 'amount_cents', null: false
+      t.string 'amount_currency', null: false
+      t.integer 'expiration', null: false
+      t.integer 'expiration_duration'
+      t.datetime 'created_at', null: false
+      t.datetime 'updated_at', null: false
+      t.index %w[organization_id code], name: 'index_coupons_on_organization_id_and_code', unique: true, where: '(code IS NOT NULL)'
```

Comment:
Layout/LineLength: Line is too long. [132/120] (https://rubystyle.guide#max-line-length)